worlds/hk/__init__.py

```diff
@@ -640,7 +642,7 @@ def __init__(self, name, advancement, code, type: str, player: int = None):
             classification = ItemClassification.progression_skip_balancing
         elif type in ("Map", "Journal"):
             classification = ItemClassification.filler
-        elif type in ("Mask", "Ore", "Vessel"):
+        elif type in ("Ore", "Vessel"):
```

Comment:
Honestly probably none of these should be filler and should all be progression_skip_balancing or similar. Vessels will matter if soul state logic is added, pale ore will matter if nail upgrades become randomized

Reply:
Probably, though right now that's not the case so they're best left not as progression. And when/if that is case, it'd probably be best to determine whether or not they are progression based off of the settings and not just always making them such.

Reply:
Fair enough I suppose